api/tests/src/db/models/factories.py

```diff
@@ -234,6 +236,27 @@ class Meta:
         sqlalchemy_session_persistence = "commit"
 
 
+class OpportunityAttachmentFactory(BaseFactory):
+    class Meta:
+        model = opportunity_models.OpportunityAttachment
+
+    attachment_id = factory.Sequence(lambda n: n)
+
+    file_location = factory.Faker("url")
```

Comment:
FYI - I need to make the ticket still, but there will be a follow-up where we'll want these to be s3 paths in the DB, and have the API presign the URLs to make them something clickable.